components/director/internal/metrics/const.go

```diff
@@ -11,4 +11,12 @@ const (
 	TenantFetcherJobName = TenantFetcherSubsystem
 	// InstanceIDKeyName missing godoc
 	InstanceIDKeyName = "instance"
+	// OrdAggregatorSubsystem missing godoc
+	OrdAggregatorSubsystem = "ordaggregator"
```

Comment:
Can you add meaningful go docs for all new constants?

Reply:
Added go doc for each constant